### PR TITLE
kev/515_impute

### PR DESCRIPTION
**Pull request by junhaow1**
# Pull Request Details

## What issue does this PR address

- IMPUTE: Constant text field only accessible when CONSTANT enabled #515

- Link to associated issue: #515 

## Checklist

Complete the check-list below to ensure your branch is ready for PR.

Flutter Style Guide: https://survivor.togaware.com/gnulinux/flutter-style.html

- [x] Screenshots included in linked issue
- [x] Changes adhere to the style and coding guideline
- [x] No confidential information
- [x] No duplicated content
- [x] No lint check errors related to your changes (`make prep` or `flutter analyze lib`)
- [ ] Pre-exisiting lint errors noted: [HERE]
- [ ] Integration test `make qtest.tmp` screenshot included in issue
- [x] Tested on device:
  - [ ] Linux
  - [x] MacOS
  - [ ] Windows
- [ ] Added two reviewers

## Finalising

Once PR discussion is complete and reviewers have approved:

- [ ] Merge dev into the branch
- [ ] Resolve any conflicts
- [ ] Add one line summary into CHANGELOG.md
- [ ] Bump appropriate version number in pubspec.yaml
- [ ] Push to git repository and review
- [ ] Merge PR into dev (gjwgit)

### CODE REVIEW

**Review by gjwgit — CHANGES_REQUESTED** [review]
Please use our standard widgets/DelayedTooltip widget so all tooltips are the same style and behaviour. Also see my note that you delted that requests

  // TODO 20240810 gjw USE CHOICE CHIP TIP

Was there a problem with that?

**Comment by junhaow1** [comment]
> Please use out standard widgets/DelayedTooltip widget so all tooltips are the same style and behaviour. Also see my note that you delted that requests
> 
> // TODO 20240810 gjw USE CHOICE CHIP TIP
> 
> Was there a problem with that?

on it

File: lib/features/impute/config.dart [hunk 1]
```diff
@@ -84,12 +85,14 @@ class ImputeConfigState extends ConsumerState<ImputeConfig> {
     super.dispose();
   }
 
+  // TODO 20240810 gjw USE CHOICE CHIP TIP
```
Comment (gjwgit) [inline lib/features/impute/config.dart]: Why not ChoiceChipTip()?

**Review by gjwgit — CHANGES_REQUESTED** [review]

**Comment by gjwgit** [comment]
Please add extra space between the Mode and the Constant chips. This gives a visual cue to the user that the Constant chip and the Constant field are conencted.

image

**Comment by gjwgit** [comment]
How come I see the default is 0 for this categoric variable. I think it should be 'Missing'

image

**Review by gjwgit — APPROVED** [review]

**Pull request merged**